core/src/Kokkos_GraphNode.hpp

```diff
@@ -32,6 +32,7 @@ static_assert(false,
 #include <Kokkos_Parallel_Reduce.hpp>
 #include <impl/Kokkos_GraphImpl_Utilities.hpp>
 #include <impl/Kokkos_GraphImpl.hpp>  // GraphAccess
+#include <impl/Kokkos_Utilities.hpp>  // intergral_constant
```

Comment:
```suggestion
#include <impl/Kokkos_Utilities.hpp>  // integral_constant
```

Reply:
What do you mean by "inlining"?
If you mean immediately-invoked lambda, that's the first thing I tried but I ran into issues with NVCC and MSVC.

Reply:
OK saw the code below.  See my comment